libs/cases/backend/case-management/src/lib/services/case.service.ts

```diff
@@ -71,6 +81,11 @@ export class XdCaseService {
 				...item,
 				overdue: Date.now() > new Date(item.dueDate).getTime(),
 			})),
+			catchError((error) => {
+				throw new BadRequestException(
+					'Failed to create Case. Please check the provided data.',
```

Comment:
```suggestion
					'Failed to create a case.',
```

The input is validated so it should be correct already

If you want to add 'real handling' you can directly print the console.log(caseData) in the next sentence